### ISSUE TITLE
Allow admins to update helper profile data

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#401]
From the public profile page, we need to offer admins the ability to modify helper data. Some data elements will only be exposed to admins (like the 'accreditation' checkbox).

See https://docs.google.com/document/d/1IknDxB-giyJSdq5WJJr7rga48GEQl-nIHg-eNBnVleI/edit# for details.

**Comment by phillipoertel** [comment]
@dottorer I read and understood the spec/requirements. Let's discuss. Question: should the "police record" feature be toggleable per circle, comparable to the "admin approves new circle members" feature? That way the additional complexity this feature introduces could be hidden from circles that don't require it.

**Comment by dottorer** [comment]
@phillipoertel  While the police record itself would be for a person uniquely, its accreditation (meaning sign off) is per circle. So we should track it on the circle/helper level. 

You are asking if we should hide the feature from a circle completely of they don't require accreditation. Depending on effort, that would be nice. Let's talk tomorrow.

**Comment by dottorer** [comment]
@phillipoertel  I slightly updated the spec doc with options to visually indicate if helpers are accredited or not. Also, per your idea I outlined a way to enable this whole feature for a circle or remove the complexity from it.

**Comment by phillipoertel** [comment]
@dottorer FYI since a user can be in more than one circle, I'll allow all his circles admins to edit his profile ...

**Comment by phillipoertel** [comment]
Todos:

- [x] circle admin can edit helper profiles (accessed through helper list)
- [x] circle admin can change passwords of all helpers
- [x] helper can only edit own profile (through "User Menu > My profile" or the helper list)
- [x] helper can only edit own password

**Comment by phillipoertel** [comment]
@dottorer deployed to staging, asking for review.

**Comment by dottorer** [comment]
@phillipoertel Tested in STG, looking very good. I like the new unified clean view with a forms edit, much simpler. Well done!

Only thing I noted was that when the admin gets the 'change password' option for a helper, he can actually set a password. Usually the admin is not supposed to know the password, but simply send the user a link to reset it himself ('request new password'). This will require the user to take action, but is safer since only the user knows the password then. If it is easy and clean to add the 'send new password request') button to the 'edit password' page, we should think about it.

Other than that, looking good and this will be a very helpful feature.

**Comment by phillipoertel** [comment]
@dottorer re password: I agree the mechanism you describe is better (and also more common). However, I simply reused the page that had been used for the password reset before. 

I just discovered that there is a real reset password page (at /reset_password) but I won't have time  to investigate why it wasn't used, verify it works correctly, change the code to use it, and remove the then old password mechanism.
